Shared/packages/OpenMobileScenePackageController.cpp

```diff
@@ -253,6 +263,48 @@ void OpenMobileScenePackageController::unpack()
   MobileScenePackage::unpack(combinedPackagePath(), unpackedDir);
 }
 
+/*!
+  \brief Creates a new scene with the default basemap and default surface.
+ */
+void OpenMobileScenePackageController::createDefaultScene()
+{
+  // obtain other required tools
+  auto basemapTool = Toolkit::ToolManager::instance().tool<BasemapPickerController>();
+  auto layerCacheTool = Toolkit::ToolManager::instance().tool<LayerCacheManager>();
+  auto dsaController = Toolkit::ToolManager::instance().tool<DsaController>();
```

Comment:
I don't really like adding dependencies on all of these tools. But maybe can't be avoided

Reply:
Will try and shuffle this around to DsaController instead